app/Services/DebugRequestActivityLogger.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace App\Services;
+
+use Illuminate\Http\Request;
+use Psr\Log\LoggerInterface;
+
+class DebugRequestActivityLogger extends AbstractRequestActivityLogger
+{
+    protected function collectRequestData(Request $request): array
+    {
+        return (
+            "Has oppen film with id" . $request->id 
```

Comment:
Should be associative array, provided string is invalid solution.

Reply:
Also should not be used request input data.

Reply:
Same problem in next logger. Read documentation, to see what important information you can extract from Request.